taskt/Core/Automation/Commands/Data Commands/GetWordCountCommand.cs

```diff
@@ -2,48 +2,48 @@
 using System.Collections.Generic;
 using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using taskt.Core.Utilities.CommonUtilities;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Data Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to get count of words from a text (string)")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to get count of words from text")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Data Commands")]
+    [Description("This command returns count of words in a string.")]
```

Comment:
Change to "This command returns of the count of all words in a string."

Reply:
Changed to "**This command returns the count of all words in a string.**"